src/api/public/apidocs-new/paths/source_project_name_package_name_file_name.yaml

```diff
@@ -0,0 +1,60 @@
+get:
+  summary: Read a specified source file
+  description: Read a specified source file and return the content
+  security:
+    - basic_authentication: []
```

Comment:
I'd say this is not needed for the `get` one. Everybody can see the content of a file, AFAIK.